### PR TITLE
RUMM-1295 Update the content for some internal logs

### PR DESCRIPTION
**Pull request by mariusc83**
### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### Review checklist (to be filled by reviewers)

- [ ] Feature or bugfix MUST have appropriate tests (unit, integration, e2e)
- [ ] Make sure you discussed the feature or bugfix with the maintaining team in an Issue
- [ ] Make sure each commit and the PR mention the Issue number (cf the [CONTRIBUTING](CONTRIBUTING.md) doc)

### CODE REVIEW

File: dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/system/BroadcastReceiverSystemInfoProvider.kt [hunk 1]
```diff
@@ -23,15 +23,17 @@ internal class BroadcastReceiverSystemInfoProvider :
     // region BroadcastReceiver
 
     override fun onReceive(context: Context, intent: Intent?) {
-        val action = intent?.action
-        when (action) {
+        when (val action = intent?.action) {
```
Comment (0xnm) [inline dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/system/BroadcastReceiverSystemInfoProvider.kt:26]: I didn't know such expression is possible in Kotlin

File: dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/system/BroadcastReceiverSystemInfoProvider.kt [hunk 2]
```diff
@@ -72,9 +74,15 @@ internal class BroadcastReceiverSystemInfoProvider :
         val level = intent.getIntExtra(BatteryManager.EXTRA_LEVEL, -1)
         val scale = intent.getIntExtra(BatteryManager.EXTRA_SCALE, 100)
 
+        val resolvedBatteryStatus = SystemInfo.BatteryStatus.fromAndroidStatus(status)
+        val resolvedBatteryLevel = (level * 100) / scale
+        sdkLogger.d(
```
Comment (0xnm) [inline dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/system/BroadcastReceiverSystemInfoProvider.kt:79]: won't it be too noisy?

File: dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/utils/WorkManagerUtils.kt [hunk 3]
```diff
@@ -18,6 +18,10 @@ import java.util.concurrent.TimeUnit
 
 internal const val CANCEL_ERROR_MESSAGE = "Error cancelling the UploadWorker"
 internal const val SETUP_ERROR_MESSAGE = "Error while trying to setup the UploadWorker"
+internal const val UPLOAD_WORKER_WAS_CANCELLED_MESSAGE =
+    "A new foreground process started. UploadWorker was canceled."
+internal const val UPLOAD_WORKER_WAS_TRIGGERED_SCHEDULED =
```
Comment (0xnm) [inline dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/utils/WorkManagerUtils.kt:23]: probably should be `UPLOAD_WORKER_WAS_SCHEDULED_MESSAGE`?

Comment (mariusc83) [reply dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/utils/WorkManagerUtils.kt:23]: yes...I know for sure I changed that but not sure what happened :)

File: dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/system/BroadcastReceiverSystemInfoProvider.kt (hunk 1 above)
Comment (mariusc83) [reply dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/system/BroadcastReceiverSystemInfoProvider.kt:26]: Was a suggestion from `AS` :)

**Review by xgouchet — APPROVED** [review]

File: dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/system/BroadcastReceiverSystemInfoProvider.kt (hunk 2 above)
Comment (mariusc83) [reply dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/system/BroadcastReceiverSystemInfoProvider.kt]: Exactly what @xgouchet said also. I removed it ;)

**Review by 0xnm — APPROVED** [review]

**Pull request merged**